float32[] velocity     #joint velocity in rad/s
float32[] stiffness    #joint stiffness in Nm/rad
float32[] damping      #joint damping (normalized, 0.7 is critically dampped)
float32[] add_torque   #additional joint torque in Nm
